packages/grid/x-data-grid/src/tests/cells.DataGrid.test.tsx

```diff
@@ -184,6 +185,37 @@ describe('<DataGrid /> - Cells', () => {
     }).toWarnDev(['MUI: The cell with id=1 and field=brand received focus.']);
   });
 
+  it('should not change the active component when the active focused cell gets out of viewport', function test() {
```

Comment:
`it('should keep the focused cell/row rendered in the DOM if it scrolls outside the viewport')`